en/user_guide/media.rst

```diff
@@ -0,0 +1,65 @@
+Media Management
+================
+
+Open Orchestra allows to manage medias upload by users in the media library panel.
+These ones can be easily adding to different blocks or contents type later.
```

Comment:
They can be easily used in other part of the website such as blocks or contents.